libs/set-algebra/set-algebra.cabal

```diff
@@ -30,7 +30,7 @@ library
         -Wincomplete-uni-patterns -Wredundant-constraints -Wunused-packages
 
     build-depends:
-        base >=4.14 && <4.19,
+        base >=4.14 && <5,
```

Comment:
Change like this requires a patch version update. Could you please bump up the version of `set-algebra` in the cabal file from `1.1.0.1` to `1.1.0.2`